eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -49,6 +49,10 @@ object LightningMessageCodecs {
     ("channelId" | bytes32) ::
       ("data" | varsizebinarydata)).as[Error]
 
+  val warningCodec: Codec[Warning] = (
+    ("channelId" | bytes32) ::
+      ("data" | varsizebinarydata)).as[Warning]
+
```

Comment:
Nit: I would move it above the `errorCodec`.

Reply:
Agreed, fixed in https://github.com/ACINQ/eclair/pull/1854/commits/ab67e7ae956ac80267dcc80506559e9a17e48bc5